src/Security/Handler/AclSecurityHandler.php

```diff
@@ -164,6 +164,8 @@ public function deleteObjectSecurity(AdminInterface $admin, $object)
     public function getObjectAcl(ObjectIdentityInterface $objectIdentity)
     {
         try {
+            // todo - remove `@var` annotation after https://github.com/phpstan/phpstan-symfony/pull/92 is released
+            /** @var MutableAclInterface $acl */
```

Comment:
It would be better to use `assert($acl instanceof MutableAclInterface)` here IMO.